src/Sylius/Bundle/AdminBundle/Resources/views/Order/Field/total.html.twig

```diff
@@ -0,0 +1 @@
+{{ data.total|sylius_money(data.currency) }}
```

Comment:
`sylius_price` should be used. `data.tota`l is amount in base currency, `sylius_money` just formats the amount, whereas `sylius_price` converts the amount from base currency to given currency and after that formats it.